src/components/timer/Pie.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export default class Pie extends React.Component {
+  constructor(props) {
+     super(props);
+    this.state = {style: {}, x: 400, y: 200, slice: (360/this.props.time), totalSlices: this.props.time};
```

Comment:
I think you should only have values you intend on mutating in the state.

Reply:
Confirming and updated.